fusor-ember-cli/app/models/openstack-deployment.js

```diff
@@ -39,6 +39,8 @@ export default DS.Model.extend(ValidatedModel, {
   undercloud_ssh_username: DS.attr('string'),
   undercloud_ssh_password: DS.attr('string'),
 
+  overcloud_deployed: DS.attr('boolean'),
```

Comment:
This makes sense to me http://stackoverflow.com/questions/1524154/naming-boolean-columns-in-rails, if the DB field is simply `overcloud_deployed` and rails respects the ?, I think it makes sense to follow that pattern in our front end for simplicity and consistency.